packages/dataviews/src/types.ts

```diff
@@ -44,7 +44,7 @@ export type Operator =
 
 export type ItemRecord = Record< string, unknown >;
 
-export type FieldType = 'text';
+export type FieldType = 'text' | 'integer';
```

Comment:
I wonder if these should be 'text' | 'integer' or  'string' or 'number' to stay as close as possible to JSON schemas.

Reply:
I went for integer because that's the type for `menu_order` according to the REST API https://developer.wordpress.org/rest-api/reference/pages/ (in terms of validation it's also more fine-grained and different than number).